plugins/channelrx/demodadsb/adsbdemodgui.h

```diff
@@ -811,6 +811,10 @@ class ModelMatch {
     {
         m_aircraftRegExp.optimize();
     }
+    
+    virtual ~ModelMatch()
```

Comment:
or "virtual ~ModelMatch = default" or better "~ModelMatch final = default"

Reply:
Right, default semantics is better :)
I'll keep virtual destructor, because class isn't final, so finalizing it's destruction less consistent.

Reply:
Btw, there are lot of warning like "warning: class with destructor marked 'final' cannot be inherited from [-Wfinal-dtor-non-final-class]". So marking destructor final causes more warnings.